#!/usr/bin/python3
def raise_exception():
    raise ValueError("This is a type exception")
